cfpb/cfpb-chart-builder
# Upgrade Capital Framework so project can use brand colors from cf-core

Removes color files from this project repo so we can use the brand colors from cf-core and other CF components.

## Additions

-

## Removals

- deleted `src/css/brand-palette.less` and `cf-theme-overrides.less` which are now obsolete

## Changes

- use updated cf-core and other CF components to get the correct colors on this project

## Testing

- check out this branch
- run `./setup.sh` to install dependencies and build assets
- view local site with `gulp watch` and confirm colors look as expected

## Review

- @marteki 
- @contolini 

[Preview this PR without the whitespace changes](?w=0)

## Screenshots


## Notes

-

## Todos

-

## Checklist
